core/src/server/subscription.rs

```diff
@@ -438,6 +438,9 @@ impl Subscription {
 		let raw = self.rx.recv().await?;
 
 		tracing::debug!("[Subscription::next]: rx {}", raw);
+
+		// clippy complains about this but it doesn't compile without the extra res binding.
```

Comment:
Ooh weird; what's the error?!

Reply:
lifetime doesn't live long enough, it may be some issue with my own serde impl I did a long time ago.

Reply:
Strange though that nothing changes at all whether the variable is there or not! I wouldn't expect any difference since you haven't given any extra type info on the variable or whatever :)